app/models/concerns/bulkrax/has_local_processing.rb

```diff
@@ -5,14 +5,24 @@ module Bulkrax::HasLocalProcessing
   # add any special processing here, for example to reset a metadata property
   # to add a custom property from outside of the import data
   def add_local
+    parsed_metadata['resource_type'] = ['ThesisOrDissertation Doctoral thesis'] if parser.is_a? Bulkrax::XmlEtdDcParser
```

Comment:
Reading note: the resource type is the key for https://github.com/scientist-softserv/britishlibrary/blob/ac0e41207d78441113c5e18647d1f23b00ebb7eb/config/authorities/resource_types.yml#L167-L168